geopandas_view/view.py

```diff
@@ -61,7 +61,10 @@ def view(
     crs="EPSG3857",
     marker_type=None,
     marker_kwds={},
-    style_kwds={},
+    style_kwds={
+        "fillOpacity": 0.5,
+        "weight": 1
+    },
```

Comment:
We shouldn't specify defaults here, but somewhere inside our function and override them via this dict. In this case, if you want to change only weight or anything else, you need to copy the whole dict. Otherwise, it defaults to folium default.

Reply:
Ah yes ! so I moved this into the function and separated out the overriding of each default style arg